src/commands/relative/create-relative.ts

```diff
@@ -33,4 +37,7 @@ export class CreateRelative {
     register.events(new RelativeAddedToRecipientUser(command.recipientUserId, command.relativeId))
     await register.flush()
   }
+
+  private static getAge = (birthDate: string): number =>
```

Comment:
This fails when using the same day, also by using time duration in a year we don't take into account leap years. We can use this instead:
```
const getAge = (birthDate: string): number => {
  const now = new Date()
  const birth = new Date(birthDate)
  let age = now.getFullYear() - birth.getFullYear()
  if (now.getMonth() < birth.getMonth() || now.getDate() < birth.getDate()) {
    age--
  }
  return age;
}
```

Reply:
By same day I mean, test with `2005-04-18` it will output 17.

Reply:
Updated!